openwisp_monitoring/notifications/admin.py

```diff
@@ -23,11 +24,11 @@ class NotificationAdmin(admin.ModelAdmin):
                 'classes': ('collapse',),
                 'fields': (
                     'actor_content_type',
-                    'actor_object_id',
+                    'actor_object_',
```

Comment:
instead of the ending underscore you may use `_link`, eg: `actor_object_link`.

Reply:
Using ending underscore helped me to render it as `Actor Object` without the need of adding any other code. Nevertheless I am shifting the code logic to `admin.py` and will implement it in `base.BaseNotificationAdmin` of Notification module, since it is more of an admin related feature.